modules/fts/pages/fts-creating-indexes.adoc

```diff
@@ -23,25 +33,59 @@ Index-creation can be performed by means of the Couchbase Web Console.
 The basic procedure is outlined in xref:fts-searching-from-the-ui.adoc[Searching from the UI].
 The current page explains the more advanced aspects of index creation, and shows how they can be managed from the console.
 
-[#accessing-and-managing-full-text-indexes]
-== Accessing and Managing Full Text Indexes
+To create Full Text Index for documents in buckets, specify the following:
 
-Full Text Indexes are different from the _Global_ indexes that are accessed under the *Indexes* tab in the left-hand navigation panel of the Couchbase Web Console.
-Full Text Indexes are accessed from the *Search* tab: left-click on this to display the *Full Text Search* panel, which contains a tabular presentation of currently existing indexes, with a row for each index.
-(See xref:fts-searching-from-the-ui.adoc[Searching from the UI] for a full illustration.) To manage an index, left-click on its row.
-The row expands, as follows:
+- Index name
+- Bucket
+-	xref:fts-creating-indexes.adoc#specifying-type-identifiers[Type Identifier]
+-	xref:fts-creating-indexes.adoc#specifying-type-mappings[Type Mappings]
+-	xref:fts-creating-indexes.adoc#specifying-fields[Fields]
+-	xref:fts-creating-indexes.adoc#creating-analyzers[Analyzers]
+-	xref:fts-creating-indexes.adoc#adding-custom-filters[Custom Filters]
+-	xref:fts-creating-indexes.adoc#date-time-parsers[Date/Time Parsers]
+-	xref:fts-creating-indexes.adoc#specifying-advanced-settings[Advanced Parameters]
 
-[#fts_index_management_ui]
-image::fts-index-management-ui.png[,820,align=left]
+To create Full Text Index for documents in scope and collections, specify the following:
 
-Three buttons are displayed:
+-	Index name
+-	Bucket
+-	xref:fts-creating-indexes.adoc#enabling-scope-collections[Enable Scope/Collection(s)]
+-	xref:fts-creating-indexes.adoc#selecting-scope[Select Scope]
+-	xref:fts-creating-indexes.adoc#specifying-type-identifiers[Type Identifier]
+-	xref:fts-creating-indexes.adoc#specifying-type-mappings[Type Mappings]
+-	xref:fts-creating-indexes.adoc#specifying-fields[Fields]
+-	xref:fts-creating-indexes.adoc#creating-analyzers[Analyzers]
+-	xref:fts-creating-indexes.adoc#adding-custom-filters[Custom Filters]
+-	xref:fts-creating-indexes.adoc#date-time-parsers[Date/Time Parsers]
+-	xref:fts-creating-indexes.adoc#specifying-advanced-settings[Advanced Parameters]
 
-* [.ui]*Delete* causes the current index to be deleted.
-* [.ui]*Clone* brings up the *Clone Index* screen, which allows a copy of the current index to be modified as appropriate and saved under a new name.
-* [.ui]*Edit* brings up the *Edit Index* screen, which allows the index to be modified.
-Saving modifications causes the index to be rebuilt.
-+
-Note that both the [.ui]*Edit Index* and [.ui]*Clone Index* screens are in most respects the same as the [.ui]*Add Index* screen, which was itself described in xref:fts-searching-from-the-ui.adoc[Searching from the UI].
+
+[#enabling-scope-collections]
+== Enabling scope/collection(s)
```

Comment:
Some mention of how the index definition has changed to accommodate scopes and collections around here would be a good idea.

Reply:
Updated the text.